src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator/BracketHelper.cs

```diff
@@ -45,12 +45,12 @@ public static bool IsBracketComplete(string query)
                         continue;
                     default:
                         {
-                            throw new ArgumentOutOfRangeException(nameof(direction), direction, "Can't process value");
+                            throw new ArgumentOutOfRangeException($"Invalid direction value. Expected values: [{TrailDirection.Open}, {TrailDirection.Close}].");
```

Comment:
Why are we trying to output those expected values? What will they actually show vs what this was showing before?